src/reducers/accounts.js

```diff
@@ -372,6 +373,12 @@ export const addHardwareAccount = (address, accountType) => async (
       })
     );
 
+    if (window.web3) {
```

Comment:
hmm is this [`init() {}`](https://github.com/makerdao/governance-dashboard/blob/91068fc939d93f2644ecc6232b436c08cfc343c9/src/index.js#L83) not triggering a `proposalsInit() {}`?

Reply:
Yes, with this set of changes, the proposal's would only render after a hardware wallet is opened. This may not be the best solution for this, an alternative is to seperate the rendering of proposal's from account management but I am not familiar enough with it to understand how coupled those two are.